mkdocs.yml

```diff
@@ -33,8 +34,8 @@ theme:
               name: Switch to light mode
     features:
         - search.suggest
-        - navigation.tabs
-        - navigation.sections
+        # - navigation.tabs
+        # - navigation.sections
```

Comment:
```suggestion
```

Reply:
I missed this comment when merging, sorry about that. Will remove those lines next time I touch this